### PR TITLE
remove lerna stuff / update contributor docs

### PR DESCRIPTION
**Pull request by dwalkr**

### CODE REVIEW

**Comment by dwalkr** [comment]
closing to kick the pull_request_target check

**Review by Enigmatical — DISMISSED** [review]
👍

**Review by Enigmatical — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,144 +6,137 @@ The following is a set of guidelines and tips for contributing to the TinaCMS an
...
+node -v
+# check yarn version, this repo ships with yarn so it should be 2.4.1
+yarn -v
+# it should show 2.4.1, you'll definitely need +2.0
+yarn install
+# build all the packages
+yarn run build
+# watch all packages
+yarn run watch
+# navigate to starter project
```
Comment (jeffsee55) [inline CONTRIBUTING.md:26]: # in a separate tab, navigate to starter project

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,144 +6,137 @@ The following is a set of guidelines and tips for contributing to the TinaCMS an
...
+Run `yarn changeset version --snapshot`
 
-   The other approach is to manually add the dependencies to the `package.json` and then run `npm run bootstrap` from the root of the repository.
+Run `yarn ci:publish-dev`
 
-3. **When I run `npm run bs` it deletes the contents of a package?**
+If you have 2FA, this will prompt you to enter you one-time code for each package you publish.
 
-   This sucks. Try running `npm run lerna -- clean` and then running `npm run bs` again.
+Run `git co -- .` This will clear out the versioning changes.
```
Comment (jeffsee55) [inline CONTRIBUTING.md:86]:
```suggestion
Run `git checkout -- .` This will clear out the versioning changes.
```

**Review by jeffsee55 — APPROVED** [review]

**Review by jeffsee55 — APPROVED** [review]

**Review by Enigmatical — APPROVED** [review]

**Pull request merged**